install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -152,6 +152,15 @@ spec:
           {{- else }}
           - istio-pilot:15011
           {{- end }}
+          {{- if $.Values.global.sds.enabled }}
```

Comment:
How does it interact with the SDS used for ingress ? Do you have any test that both can be used at the same time ?

Reply:
ingress sds uses a separate flag

Reply:
See #15268 - this will no longer be needed.